public/lang/fr.ini

```diff
@@ -58,14 +58,16 @@ lang_sources_refreshed=Les sources ont été rafraichies
 lang_error_session_expired=Votre session a expiré
 lang_error_add_source=Erreur lors de l'ajout de la source :
 lang_error_delete_source=Erreur lors de la suppression de la source :
-lang_error_load_tags=Erreur lors du chargement des balises :
-lang_error_unknown_tag=Étiquette inconnue :
+lang_error_load_tags=Erreur lors du chargement des tags :
+lang_error_unknown_tag=Tag inconnu :
 lang_error_unknown_source=Identifiant de source inconnu :
 lang_error_mark_items=Impossible de marquer tous les éléments visibles
+lang_error_mark_item=Erreur lors du marquage lu/non lu :
+lang_error_star_item=Erreur lors de l'ajouter/suppression du favori:
```

Comment:
Should not this use `’` instead of `'` and have a space in front of `:`?

Reply:
And where there is a space before punctuation, should it not be a thin non-breaking space?

Reply:
Hmm, looking at http://unicode.org/udhr/n/notes_fra.html, the thin non-breaking space is only used before `!` and `?`, `:` is preceded by standard non-breaking space (`U+A0`).